app/views/transition_landing_page/_buckets.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="landing-page__section"
   data-analytics-ecommerce
   data-ecommerce-start-index="1"
-  data-list-title="Brexit landing page: <%= guidance_header %>"
+  data-list-title= <%= t("transition_landing_page.data_list_title", header: guidance_header) %>
```

Comment:
Does this need to be wrapped in quotes and the space after the `=` removed, otherwise the HTML may not be valid?

Reply:
Good spot, this was an oversight by me!